src/nodes/pmrem/PMREMUtils.js

```diff
@@ -214,7 +214,7 @@ export const textureCubeUV = /*@__PURE__*/ Fn( ( [ envMap, sampleDir_immutable,
 
 } );
 
-const bilinearCubeUV = /*@__PURE__*/ Fn( ( [ envMap, direction_immutable, mipInt_immutable, CUBEUV_TEXEL_WIDTH, CUBEUV_TEXEL_HEIGHT, CUBEUV_MAX_MIP ] ) => {
+export const bilinearCubeUV = /*@__PURE__*/ Fn( ( [ envMap, direction_immutable, mipInt_immutable, CUBEUV_TEXEL_WIDTH, CUBEUV_TEXEL_HEIGHT, CUBEUV_MAX_MIP ] ) => {
```

Comment:
The error I'm getting is here: only the first two arguments are defined, so it crashes when it tries to read `mipInt_immutable`. My other comment shows where this is being called. I thought I had verified this kind of functional programming works in your TSL editor, but maybe something is subtly different? Am I hitting a real limitation of TSL here, or am I just doing something stupid? Any guidance @sunag?

Reply:
It seems to be related to the first argument of TSL function, I think that if you change of `( envMap, sampleDirection ) 
 =>` to `( [ envMap, sampleDirection ] ) =>` it will work.

The second parameter in JS function is reserved for `builder`, where you can access the related material, object and geometry. For example: `( [ envMap, sampleDirection ], builder ) =>`.

It is also possible to pass parameters as objects for example: `( { envMap, sampleDirection } ) =>` and call using `bilinearCubeUV( { envMap, sampleDirection  } )`, can be useful for functions with many parameters.